en/docs/insights/programmatic-access-choreo-insights-api.md

```diff
@@ -0,0 +1,60 @@
+# Programmatic Access to Insights API
+
+The insights page in the Choreo Console fetches data from the Insights API exposed over the Internet. Occasionally, some users might need to fetch these data from external systems for various use cases. One such use case can be to implement their own dashboards based on these data.
+
+Choreo provides a secure and user-friendly method to achieve this requirement. The steps are outlined below.
+
+## Become a Member of the Choreo System Organization
+
+1. Sign in to [https://devportal.choreo.dev](https://devportal.choreo.dev) using your credentials.
+
+2. Visit [https://devportal.choreo.dev/choreosystem](https://devportal.choreo.dev/choreosystem) to access the developer portal of Choreo System organization.
+
+    If you are accessing the Choreo System organization’s developer portal for the first time, you will need request access to the organization by clicking on the **Request Access** button.
+
+    ![Request-Access](../assets/img/insights/request-access.png){.cInlineImage-full}
+
+3. Once access has been requested, you will be automatically added as a member of the Choreo System organization.
+
+    Verify this by inspecting the organization drop-down(Choreo System organization should be visible).
+
+    ![Choreo-System-Home](../assets/img/insights/choreo-system-home.png){.cInlineImage-full}
+
+## Create an Application & Generate Keys
+
+1. Navigate to the Applications' menu to create a new application in the Choreo System organization.
+
+    ![Create-Application](../assets/img/insights/create-application.png){.cInlineImage-full}
+
+2. Provide a suitable name and create the application.
+
+    ![Create-Application-Wizard](../assets/img/insights/create-application-wizard.png){.cInlineImage-half}
+
+3. Subscribe to the **Insights** API using the newly created application.
+
+    ![Subscribe-API](../assets/img/insights/subscribe-api.png){.cInlineImage-half}
```

Comment:
Align the image within a bullet point. This comment applies to the next two images as well.

Reply:
Shall we describe how it is done?

e.g., "To subscribe to the new API via the newly created application, click **Subscriptions** in the left navigator, and then click **Add** for the **Insights** API."